tests/Backend/Snowflake/TableDefinitionOperationsTest.php

```diff
@@ -2002,6 +2002,44 @@ public function testAddTypedColumnOnNonTypedTable(): void
         }
     }
 
+    public function testAddNullableColumn(): void
+    {
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
+
+        $tableDefinition = [
+            'name' => 'my-new-table-non-typed',
+            'primaryKeysNames' => [],
+            'columns' => [
+                [
+                    'name' => 'id',
+                    'definition' => [
+                        'type' => 'INT',
+                        'nullable' => false,
+                    ],
+                ],
+            ],
+        ];
+
+        $tableId = $this->_client->createTableDefinition($bucketId, $tableDefinition);
+
+        // add row
+        $csvFile = $this->createTempCsv();
+        $csvFile->writeRow(['id',]);
+        $csvFile->writeRow([1,]);
+        $this->_client->writeTableAsync($tableId, $csvFile);
+
+        try {
+            $this->_client->addTableColumn($tableId, 'fail', [
+                'type' => 'VARCHAR',
+                'nullable' => false,
+            ]);
+            $this->fail('Should throw ClientException');
+        } catch (ClientException $e) {
+            $this->assertSame('Non-nullable column "fail" cannot be added to non-empty table "my-new-table-non-typed" unless it has a non-null default value.', $e->getMessage());
```

Comment:
Ještě mě napadá - neměl by být i test, že to na non-empty tabulce projde pokud bude nullable?

Reply:
Nevím přijde mě to zbytečné, ale asi takový test teď nemáme co sem sa tak díval.

Reply:
Jo, fér. Tím, že je to exception ze SNFLK a ne přímo náš use case tak asi whatever :)